package.json

```diff
@@ -14,22 +14,20 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "4.1.3",
-    "@angular/common": "4.1.3",
-    "@angular/compiler": "4.1.3",
-    "@angular/compiler-cli": "4.1.3",
-    "@angular/core": "4.1.3",
+    "@angular/animations": "^4.2.4",
```

Comment:
I see that this was changed from specific version numbers to version ranges. 
I believe that we changed to specific versions to avoid random failures when underlying dependencies changed. 
Is there a reason this is changing that back?

Reply:
When upgrading, by default it will change it to the ranges.  I suppose we can change it to fixed or "~" if needed.

Reply:
I think fixed makes more sense